java/src/jmri/jmris/json/Bundle.properties

```diff
@@ -9,7 +9,7 @@ MenuItemStartServer=Start JMRI JSON Server
 PreferencesItem=NETWORKSERVICES
 PreferencesItemTitle=JSON Server
 PreferencesTabTitle=JSON Server
-PreferencesTooltip=Set preferences for the JSON Server
```

Comment:
This should be lower case, not upper case, "Preferences" does not refer to a window or other user interface element.